### PR TITLE
feat: add builder ktx for FirebaseRemoteConfigSettings

### PR DESCRIPTION
**Pull request by rosariopfernandes**
Add Kotlin Extensions:
1. Building `FirebaseRemoteConfigSettings` objects 
```kotlin
val configSettings = remoteConfigSettings {
    setMinimumFetchIntervalInSeconds(3600)
    setFetchTimeoutInSeconds(60)
}
```

2. Setting Config Settings on a FirebaseRemoteConfig object
```kotlin
remoteConfig.setConfigSettingsAsync {
    setMinimumFetchIntervalInSeconds(3600)
    setFetchTimeoutInSeconds(60)
}
```

### CODE REVIEW

**Comment by rlazo** [comment]
Thanks for your contribution @rosariopfernandes! I like the proposal to extend the `FirebaseRemoteConfigSettings.Builder`. We could also add the missing getters in Java to use properties. 

I'm not totally sold on the second proposal. I think we could get most of the gains from just implementing the first part.

```
remoteConfig.setConfigSettingsAsync(remoteConfigSettings { 
    minimumFetchIntervalInSeconds = 3600
    fetchTimeoutInSeconds = 60
})
```

**Comment by rosariopfernandes** [comment]
Thanks for the review @rlazo . I've removed the second proposal and added the missing Java getters.

File: firebase-config/ktx/ktx.gradle [hunk 1]
```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation project(':firebase-config')
     implementation project (':firebase-abt')
     implementation 'androidx.annotation:annotation:1.1.0'
+    implementation 'com.google.android.gms:play-services-tasks:17.0.0'
```
Comment (rlazo) [inline firebase-config/ktx/ktx.gradle]: Is this dependency needed? The only new import is in `RemoteConfig.kt` and doesn't seem related to that library.

Comment (rosariopfernandes) [reply firebase-config/ktx/ktx.gradle]: It was being used for the second proposal. I forgot to remove the dependency when I removed the function. But it's done now, thanks

**Comment by rlazo** [comment]
Hi @rosariopfernandes, thanks a lot for your contribution to Firebase! We've reviewed your proposal internally and it's been approved as-is. It will be merged into `master` as soon as the pending checks pass.

**Comment by rosariopfernandes** [comment]
@rlazo Thanks!

I wonder if the changes from #876 are somehow affecting the `new-smoke-tests` job here.

**Comment by rlazo** [comment]
> @rlazo Thanks!
> 
> I wonder if the changes from #876 are somehow affecting the `new-smoke-tests` job here.

I've seen that error before, so I'm not sure whether it's related. We're looking into it 😄

**Comment by rlazo** [comment]
As the failure isn't cause by the PR, we're merging it. Thanks again @rosariopfernandes !
